### ISSUE TITLE
Navigating from TabView child elements

### ISSUE THREAD

**Issue by pjpauwels** [NativeScript/nativescript-angular#492]
I'm calling a route navigation from within a tabview (simplified version below) which throws an Exception.

```
<TabView #tabview>
    <StackLayout *tabItem="{title: 'Tab 1'}">
        <Label text="A label" (tap)="nav()"></Label>
    </StackLayout>
</TabView>
```

```
nav() { this.router.navigate(['/contact-detail', 1]);}
```

This throws the following error:

> Unhandled Exception
> com.tns.NativeScriptException:
> Calling js method onDestroyView failed
> 
> TypeError: Cannot read property 'typeface' of null
> File: '/data/data/my.package.name/files/app/tns_modules/ui/tab-view/tab-view.js, line: 355, column: 42

When I place the Label outside the TabView it all works fine.

**Comment by tsonevn** [comment]
Hi @pjpauwels,

I reviewed the given scenario with our `RouterExtensions` and TabView, however I was unable to reproduce issues while navigating through code. You could also review the [Navigation article](https://docs.nativescript.org/core-concepts/angular-navigation#navigation-options) For you convenience  I am attaching sample code.

Sample:

```
import { Component } from "@angular/core";
import { NativeScriptRouterModule, RouterExtensions } from "nativescript-angular/router";

@Component({
    selector: 'basic-tab-view-component',
    templateUrl: 'tab-view/basic-tab-view/basic-tab-view.component.html'
})

export class BasicTabViewComponent {
   public tabindex = 1;

    constructor(private routerExtensions: RouterExtensions){

    }
   public nav(){
    this.routerExtensions.navigate(["/firstPage"]);
   }
}

```

Let me know whether this help.

**Comment by pjpauwels** [comment]
Hi @tsonevn,

Thank you for taking a look at this ticket.
I have tried the routerExtension way as well and have not found any difference in what happens.

However I have finally found my issue now. It seems it is not related to navigating at all. After I added a font-family and font-size to the TabView everything is working again. (Setting it either inline style of through a class fixes it)

I am not sure if this is specific to my device and if it is common at all. I am using a OnePlus2 with OxygenOS 3.1.0.

Somehow when no font-family or size is causing some nullPointer issues in `TabViewStyler.setFontInternalProperty`. Might be something to look into null checking though.
